department-of-veterans-affairs/va.gov-cms
# VACMS-4513: Disallow wysiwyg on core tax description field.

## Description
See #4513

## Testing done
Visual / behat

## Screenshots
![image](https://user-images.githubusercontent.com/2404547/112357744-306f6a80-8ca6-11eb-851b-3c90dc95f8d3.png)

## QA steps

As user _uid_ with _user_role_
As an administrator, go to `taxonomy/term/94/edit` and visually verify that wysiwyg is no longer attached to description field.

### Definition of Done


### Select Team for PR review


### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
